.github/workflows/test.yml

```diff
@@ -28,9 +28,22 @@ jobs:
           pip install pylint
           if [ -f setup.py ]; then pip install -e .; fi
           pip install transformers
+
+          #Install miniconda
+          wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+          bash Miniconda3-latest-Linux-x86_64.sh -b -u
       - name: Lint with PyLint
         run: |
           pylint src --rcfile .pylintrc
+      - name: Check installation of model requirements on isolated conda env
```

Comment:
Move after unittest, since this is not a unit test

Reply:
done